style.css

```diff
@@ -1 +1,190 @@
-/*  Add your styling here */
+/* Add your styling here */
+body{
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
+.logoup{
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    width: 30%;
+}  
+
+/* cart icon */
+h3{
+    color: #ffffff;
+    float: right;
```

Comment:
if you want to align text to the right, you can use `text-align` or otherwise use justify-content if this was flexbox